ui/components/app/edit-gas-popover/edit-gas-popover.component.js

```diff
@@ -85,7 +87,7 @@ export default function EditGasPopover({
   } = useGasFeeInputs(defaultEstimateToUse, transaction, minimumGasLimit, mode);
 
   const [showAdvancedForm, setShowAdvancedForm] = useState(
-    !estimateToUse || hasGasErrors,
+    !estimateToUse || hasGasErrors || !networkAndAccountSupport1559,
```

Comment:
Is this right? Shouldn't we want to show the advanced options in the edit display right off the bat if the user is not on a 1559 network?

Reply:
> Shouldn't we want to show the advanced options in the edit display right off the bat if the user is not on a 1559 network?

Yes, exactly. So if there is no network or account support, then `showAdvancedForm` will be set to true.

Reply:
Oh lol right...